api/tools/src/main/java/org/pmiops/workbench/tools/RunGenomicExtractionWorkflow.java

```diff
@@ -0,0 +1,236 @@
+package org.pmiops.workbench.tools;
```

Comment:
might be good to add a comment that this is for cmd tooling, not part of core RWB code

Reply:
Do you mean for this tool, or for the "tools" package generally?  All of the tools in "tools" are like that.